app/controllers/items_controller.rb

```diff
@@ -661,4 +749,23 @@ def enforce_versioning
     render status: :forbidden, text: 'Object cannot be modified in its current state.'
     false
   end
+
+  # ---
+  # Dor::Workflow utils
+
+  def dor_accession_error?(object, task)
```

Comment:
Also slightly modified in the APO PR at https://github.com/sul-dlss/argo/blob/issue76-apo-nice/app/controllers/workflow_service_controller.rb